/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
/apps/brussel/RL8/skylake/software/SciPy-bundle/2022.05-foss-2022a/lib/python3.10/site-packages/numpy/lib/nanfunctions.py:1584: RuntimeWarning: All-NaN slice encountered
  result = np.apply_along_axis(_nanquantile_1d, axis, a, q,
Traceback (most recent call last):
  File "/vscmnt/brussel_pixiu_data/_data_brussel/vo/000/bvo00012/vsc11137/source2suffering/main.py", line 546, in <module>
    ds_pf_gs = gridscale_emergence(
  File "/vscmnt/brussel_pixiu_data/_data_brussel/vo/000/bvo00012/vsc11137/source2suffering/gridscale.py", line 416, in gridscale_emergence
    pk.dump(ds_pic,f)                
MemoryError
